res/spray/spray.obj

```diff
@@ -0,0 +1,17364 @@
+# Blender 4.2.0 Alpha
```

Comment:
Aa far as I understand, this file is not used anymore, as there is no obj loader code. Remove this file.

Reply:
same here, leave this file as is